1 HW_Standard/HW_Grovyle.yaml

```diff
@@ -0,0 +1,22 @@
+species: Grovyle
+ingamename: Sephiroth
+displayname: Sephiroth
+setname: Christmas
+tags: [christmas]
+item: Life Orb
+ability: No Guard
+moves:
+    - [Leaf Blade]
+    - [Draco Meteor]
+    - [Cosmic Power]
+    - [Nasty Plot]
+gender: [m]
+happiness: 255
+biddable: True
+rarity: 1.0
+ball: [Dusk]
+shiny: True
+nature: Naughty
+ivs: 31
+evs: {hp: 0, atk: 252, def: 0, spA: 252, spD: 0, spe: 0}
```

Comment:
A bit too strong, I would remove LO and reduce the IVs to 15. if this is intended to be a halloween set you could switch leaf blade to night slash and remove spatk investment instead of removing LO. I would also think a fire attack to refernce niethlhiem/flare would be better than cosmic power.